vendor/bat-native-ledger/src/bat_client.cc

```diff
@@ -649,6 +649,22 @@ void BatClient::GetAddressesForPaymentIdCallback(
   ledger_->SetAddresses(addresses);
 }
 
+void BatClient::ShouldCreateWallet() {
+  const std::string payment_id = ledger_->GetPaymentId();
+  const uint64_t stamp = ledger_->GetBootStamp();
```

Comment:
nits:

`std::string`, `uint64_t` should be `auto`

Reply:
what is benefit of it? is this maybe best practice?

Reply:
Google/Chromium coding standards for type names which are noisy, obvious, or unimportant

Reply:
ok sounds good

Reply:
fixed